scripts/train/train.py

```diff
@@ -104,7 +104,7 @@ def build_composer_model(model_cfg: DictConfig,
 
 
 def build_composer_peft_model(
-        model_cfg: DictConfig, lora_cfg: DictConfig,
+        pretrained_model_name_or_path: str, lora_args: Dict[str, Any],
```

Comment:
can you rebase on main? I think some of the changes in here are already merged via one of your other prs.

Reply:
rebased